apps/client/src/services/utils.ts

```diff
@@ -297,6 +297,54 @@ function isHtmlEmpty(html: string) {
     );
 }
 
+function formatHtml(html: string) {
```

Comment:
There is already a formatting function in `ReadOnlyCodeTypeWidget`. Is it similar, can we use that one instead?